src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1161,175 +1161,175 @@ f1_co_directors,45,1994,Line 45,,,,
+f1_comp_balance_db,65,2004,Derivative Instrument Assets - Hedges (176),reported_value,derivative_instrument_assets_hedges,,
+f1_comp_balance_db,66,2004,(Less) Long-Term Portion of Derivative Instrument Assets - Hedges (176,reported_value,derivative_instrument_assets_hedges_long_term,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,67,2004,Total Current and Accrued Assets (Lines 34 through 66),calculated_value,current_and_accrued_assets,,
+f1_comp_balance_db,68,2004,DEFERRED DEBITS,header,,,
+f1_comp_balance_db,69,2004,Unamortized Debt Expenses (181),reported_value,unamortized_debt_expense,,
+f1_comp_balance_db,70,2004,Extraordinary Property Losses (182.1),reported_value,extraordinary_property_losses,,
+f1_comp_balance_db,71,2004,Unrecovered Plant and Regulatory Study Costs (182.2),reported_value,unrecovered_plant_and_regulatory_study_costs,,
+f1_comp_balance_db,72,2004,Other Regulatory Assets (182.3),reported_value,other_regulatory_assets,,
+f1_comp_balance_db,73,2004,Prelim. Survey and Investigation Charges (Electric) (183),reported_value,preliminary_survey_and_investigation_charges,,
+f1_comp_balance_db,74,2004,Preliminary Natural Gas Survey and Investigation Charges 183.1),reported_value,preliminary_natural_gas_survey_and_investigation_charges,TRUE,"This charge was previously combining nat gas and other charged as evidenced by the (183.1, 183.2) and now only showing (183.1). There is no uncombined xbrl factoid for gas even though there is an uncombined xbrl factoid for other."
+f1_comp_balance_db,75,2004,Other Preliminary Survey and Investigation Charges (183.2),reported_value,other_preliminary_survey_and_investigation_charges,,
+f1_comp_balance_db,76,2004,Clearing Accounts (184),reported_value,clearing_accounts,,
+f1_comp_balance_db,77,2004,Temporary Facilities (185),reported_value,temporary_facilities,,
+f1_comp_balance_db,78,2004,Miscellaneous Deferred Debits (186),reported_value,miscellaneous_deferred_debits,,
+f1_comp_balance_db,79,2004,Def. Losses from Disposition of Utility Plt. (187),reported_value,deferred_losses_from_disposition_of_utility_plant,,
+f1_comp_balance_db,80,2004,"Research, Devel. and Demonstration Expend. (188)",reported_value,research_development_and_demonstration_expenditures,,
+f1_comp_balance_db,81,2004,Unamortized Loss on Reaquired Debt (189),reported_value,unamortized_loss_on_reacquired_debt,,
+f1_comp_balance_db,82,2004,Accumulated Deferred Income Taxes (190),reported_value,accumulated_deferred_income_taxes,,
+f1_comp_balance_db,83,2004,Unrecovered Purchased Gas Costs (191),reported_value,unrecovered_purchased_gas_costs,,
+f1_comp_balance_db,84,2004,Total Deferred Debits (lines 69 through 83),calculated_value,deferred_debits,,
+f1_comp_balance_db,85,2004,"TOTAL ASSETS (lines 14-16, 32, 67, and 84)",calculated_value,total_assets,TRUE,A combination of assets not reflected in xbrl
```

Comment:
Looking at [the XBRL taxonomy for this table](https://xbrlview.ferc.gov/yeti/resources/yeti-gwt/Yeti.jsp#tax~(id~8*v~72)!con~(id~78077)!net~(a~143*l~35)!lang~(code~en)!path~(g~3709*p~0_0_0_0_4)!rg~(rg~4*p~1)) it appears to me that the "TOTAL ASSETS" line should probably map to `assets_and_other_debits` which is described as: "TOTAL ASSETS (lines 14-16, 32, 67, and 84)"

Reply:
I'm not sure about that. I already mapped the `assets_and_other_debits` factoid to the row literal `TOTAL Assets and Other Debits (Enter Total of lines 10,11,12,22,52,68)`. The "TOTAL ASSETS" row seems to be something different. Also I noticed that there were no leftover xbrl factoids.

Reply:
I think it should be mapped to **both** of those -- it's the big total at the end of the whole page in both cases.